### PR TITLE
Add support for filtering by a classname longer than twenty characters and multiple classnames

### PR DESCRIPTION
**Pull request by igpaec**
PR to address Issue #183

Example with multiple class names all of them bigger than twenty characters long.
image

### CODE REVIEW

File: nautobot_data_validation_engine/models.py [hunk 1]
```diff
@@ -318,13 +319,13 @@ def clean(self):
 class DataCompliance(PrimaryModel):
     """Model to represent the results of an audit method."""
 
-    compliance_class_name = models.CharField(max_length=100, blank=False, null=False)
+    compliance_class_name = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=False, null=False)
     last_validation_date = models.DateTimeField(blank=False, null=False, auto_now=True)
     content_type = models.ForeignKey(ContentType, on_delete=models.PROTECT, blank=False, null=False)
     object_id = models.CharField(max_length=200, blank=False, null=False)
```
Comment (glennmatthews) [inline nautobot_data_validation_engine/models.py:325]: Should `object_id` and `validated_object_str` and `validated_attribute_value` also be increased? Honestly not clear on why these are limited at all.

Comment (igpaec) [reply nautobot_data_validation_engine/models.py]: I have made the change for all the CharFields. As to why they are limited, I am not sure. Maybe to make it compatible with MySQL which does not have unlimited VARCHAR?

File: nautobot_data_validation_engine/forms.py [hunk 2]
```diff
@@ -297,8 +299,8 @@ class DataComplianceFilterForm(BootstrapMixin, forms.Form):
     """Form for DataCompliance instances."""
 
     model = DataCompliance
-    compliance_class_name = forms.CharField(max_length=20, required=False)
-    validated_attribute = forms.CharField(max_length=20, required=False)
+    compliance_class_name = MultiValueCharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
+    validated_attribute = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
```
Comment (glennmatthews) [inline nautobot_data_validation_engine/forms.py:303]: Should this also be a `MultiValueCharField`?

Comment (igpaec) [reply nautobot_data_validation_engine/forms.py:303]: Yes

**Review by gsnider2195 — APPROVED** [review]

**Pull request merged**